prisma/migrations/migrate.lock

```diff
@@ -6,4 +6,6 @@
 20200624211600-initial
 20200625141833-remove-language-learning-unique-constraint-on-user-id
 20200702185801-fix-issues
-20200805064655-add-published-at-to-posts
\ No newline at end of file
+20200805064655-add-published-at-to-posts
+20200813085658-add-post-comment
+20200813092954-add-post-comment
```

Comment:
I don't see `20200813085658-add-post-comment` directory, does that mig need to be removed from this file?

Reply:
Great catch! ✅